### PR TITLE
Refactor dir notebooks

### PR DESCRIPTION
**Pull request by rocco8773**
1. rename dir `notebooks` -> `tutorials_and_notebooks`.  I thought this would be more appropriate so presenters can also upload a pdf copy of their slides along side their notebook
2. add a README to the `tutorials_and_notebooks` dir

### CODE REVIEW

File: tutorials_and_notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+For the pre- Hack Week Python tutorials please visit the
+[hack-week-python-intro](https://github.com/PlasmaPy/hack-week-python-intro)
+repository.
+
+## For Presenters
+
+To upload your slides and notebook to this repository please open a pull
+request and we will be happy to merge.  If you are unfamiliar with this
+process or prefer an alternate method, then please email a pdf copy of
+your slides and/or your notebook to <hack@plasmapy.org> and we will upload
```
Comment (namurphy) [inline tutorials_and_notebooks/README.md:16]:
```suggestion
your slides and/or your Jupyter notebook to <hack@plasmapy.org> and we will upload
```

**Review by namurphy — APPROVED** [review]
Looks good!  One potential optional clarification.

File: tutorials_and_notebooks/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Tutorials & Notebooks
+
+This directory contains slides and notebooks associated with the presentations
```
Comment (namurphy) [inline tutorials_and_notebooks/README.md:3]:
```suggestion
This directory contains slides and Jupyter notebooks associated with the presentations
```

**Comment by rocco8773** [comment]
@namurphy Thank you for the quick look over.

**Pull request merged**